### PR TITLE
Qiime

### PR DESCRIPTION
**Pull request by lleroi**
solve some issues from #442

### CODE REVIEW

File: tools/qiime/split_libraries.xml [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-	</requirements>
-	<command>
-	
+    <description>Split libraries according to barcodes specified in mapping file</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <expand macro="requirements" />"package">qiime</requirement>
+    </requirements>
```
Comment (nsoranzo) [inline tools/qiime/split_libraries.xml]: The old requirement has not been removed properly.

File: tools/qiime/make_otu_table.xml [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-		<requirement type="package">qiime</requirement>
-	</requirements>
-	<command>
+    <description>Make OTU table</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <expand macro="requirements" />ckage">qiime</requirement>
+    </requirements>
```
Comment (nsoranzo) [inline tools/qiime/make_otu_table.xml]: The old requirement has not been removed properly.

**Comment by bgruening** [comment]
@lleroi are you working on this branch or should I merge it :)
It contains some very nice fixes and I think it would be nice to have it in our main working branch.

**Comment by lleroi** [comment]
@bgruening you can merge this branch. I will finish to fix #442 issues tomorrow and make a new PR :)

**Comment by bgruening** [comment]
Wonderful!

**Pull request merged**